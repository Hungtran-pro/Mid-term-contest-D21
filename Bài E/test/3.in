10
886 8
ggebf?db?eahfeddcgbbac?ddcheecgbadefbgc?bgfhdfa?acchceh?eec??fedha?gh?gdehab?egadedbffedgagbeg?gcfdggegd?cf?h?cbhceaee?acfchedcffdhcef?badca?hfce?dbcchhahbhag?adgfchddebdbgaaeeaagddhhd?gffg?aaaggcchgeabheffgefdedcbega?aahab?cbb?fabe?gcgbcadacf?bdhhdhddbhghb?g??gbd??bagacddgaghc??gcgdghghedcecbhhbgbcddhfacgdgdd?ah?baghdadhdbeaebgbhhgccheghdfa?abbacacgdee?efbgaeahfebef?ggfcdddgd?g?decfbghecebf?bfacadb??baggh?ehhcedcdgfec?aehafddehf?ahdc?fgaeefaahaafdcaadhgheg?dgehahaeggfdaecgdgbhccccfchfcdcbhh?b?chhabgcabeddahhcabegafaghdd?age?ddgfadbaa?ghahcggbbcega?edabaaedeeeb?baf?hc?fgcbdgebg?ced?g?caecba?dcgh?cebdahebchgbgfg?gfbag?b?dfb?hfagdecdgfgbdfefcdaecaahdceee?babbbdbheggaccccdffdacddb?bhgegc?ghbhad?ccbgd?bbbghgdc?chddfeeec?egcdefbdeadhdccbccd?baeebadce?dbghgacdacgfg?bfdddchh?d?hfdabcggaahffeecfggbdf?hfbdfhdadaa?agg?ffgacc?hgecgceeaghchf?debbegcfdhfccbacbhe?gha?f??ehebgfbcgfdddc?a?
857 10
eigh?h?gejcbcfbfi?idaega?aja?ce??ieeaigebebgjhdhdbghejbcdgdadfjecfgf?ihj?bac?g??c?acadgaaiiigf?jfcidcgceejafhhi?fifcgj?ihdjajaahjhghgdgecifhefged?eicijdejebaffidhbeh??eacdgedh?hc?echdbfaajdedcjfdhdieechgdaidadehbbdjbbfhbdiigfhajbb?bijf?ig??bageifefggdedehceijbgd?d?cheeejeffcjcagi???ea?c?gcjfjhefai???bggjdfjhjiee?daffaggbeeidfgi?ecgaecbcbb?jjibafjeaejdjfj?ccfgabgaebe?dgjhfhbddccceedcj?idefdechi?ebceaha?bgcbc?ggfefiibdbgahijhe?fiiidbjeiaabcea?gfcgf?gaifaicehggbh?ihiacjfhccaih?f?ejdgcgacdhdhag?bifaice?fij?cbaiadahfiaajdbehjdhbacaidfdffe?ajcebgafegfeigiicgijigeiibei?ieef?cbhafcbcbadgicg?bahdhfegeccdjfjcjhfeadjcjbijdibidegfhgf?hijhggjcbaf?j?iagdfgiigbiggebdbggegefh?bfiiffa?chb?hf?ajiachcjj?jdiheejdie?ihhjgaacidci?aicedbdaeijcfbhbiecijc?ecifdgcijcdcfbiaedjhhhdifjg?gbfgecaa?fgfgijdeaeagijh?eecg?cjiaeijcdjc?dbbf?baeihccedebfdii?h?afghbbgjcdagbe?dagihajh
596 8
fhbga?gh?gceahfhe?bfahef?fb?bdagdc?cbbgbfceheageffcbfhcfcdfdgfad?fcddafe?hggf?ceec?e?fffeadbhfedgh?cggef?cdcbfh?cg?db?ddghgh?ceeeebe?f?beaechg?fdbhaedcdehhgeahfe?decafgdga??c?gdffe?dd??heabhagegafe???egadfeadf?aafgefcaghhhf?eefhagfdh??dhhegfchece?g?ca?bbggfdf?hbhgfhc?dd?hcfhgfdhf?ccf?afbdfbcdeggahbeddeabbhfac?g?gaecaah?ahhb?gdbfbccad?cfefahhgcee?gbb?hfcgda?hehgbfee?fchhchchdch?f?eggbhhhgd?cagehhdh?e?ddbbgghghfabhcagfef?dec?bgbfg?heg?f?adgfbdgbdf?cbhhdfeeafccec?hefbd?edbcehh?c?bgcdehehhbdgehhaeeg?aechc?fbefgagdfbcggdggeg?ahdbahcbchade?adgbf?bgffacaahffcecdhbgcdagcdcc?hbf?fcfbg?gbfe?gchfg?ef
547 9
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaadddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd????????????????????????????????????????????????????????????eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee????????????????????????????????????????????????????????????bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb???????
862 10
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh??????????????????????????????????????????????????????????????????????????????????????iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeegggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg??
848 6
fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee??
749 10
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff??????????????????????????????????????????????????????????????????????????iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiihhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee?????????
747 7
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa?????
598 9
??????????????????????????????????????????????????????????????????bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg??????????????????????????????????????????????????????????????????eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeecccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccciiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg????
615 7
???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
