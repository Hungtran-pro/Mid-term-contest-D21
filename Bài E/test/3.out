3
3
3
20
28
141
18
1
33
87
